### ISSUE TITLE
Jail time and drop settings

### ISSUE THREAD

**Issue by xorex** [dmustanger/7dtd-ServerTools#159]
As far as I understand, by default the player is in jail for 60 minutes or until 200 coins are paid.

It would be nice to change this value in the settings or when sent to prison by the console command.
Example:

> Jail add 76561191234567891 **900**

and config file
> <Tool Name="Zones" **time="180"** Enable="False" Kill_Enabled="False" Jail_Enabled="False" Kick_Enabled="False" Ban_Enabled="False" Zone_Message="False" Set_Home="False" Days_Before_Log_Deleted="5"  />

If possible, setting in **zones.xml** so that when entering the zone, the player dropping all his stuff.
Example:
> <zone corner1="100,-56,100" corner2="-100,200,-100" **dropall="true"** entryMessage="You have entered the Market" exitMessage="You have exited the Market" response="say {PlayerName} has entered the market" reminderNotice="" circle="false" PvE="true" noZombie="true" />

maybe a file **jail_allow_stuff.xml** with a whitelist of things that were not thrown away when I went to prison was amazing, but it’s probably too difficult.

[off-topic] _/playerlist or /list commands does not work on ST 12.95_

**Comment by suL33T** [comment]
Where is the setting for the phrase pay 200 coins? Is it next to the other jail phrases? What's the number please?

**Comment by ObsComp** [comment]
He setup a custom command with custom commands that costs 200 coins and runs the jail removal on them. That is not built in to ST but available with the custom commands tool.

It was setup as a generic value for now but might be changed at a later date. Now that the base game code is not changing as radically, I will start refining more tools for more options.

I did some testing with /playerlist in 13.0, it is working as intended and /friendlist has also been fixed.
